packages/scrollable-image/dev/local-build.js

```diff
@@ -0,0 +1,30 @@
+import fs from 'fs'
+import path from 'path'
+import ScrollableImage from '../src/index'
+
+// const data = [
+//  'https://static01.nyt.com/newsgraphics/2016/08/14/men-100-meters-bolt-horizontal/09c0dfe010da583c01f23709a11f6153e10cbb7b/bolt-100m-race-a3698x450.jpg',
+// ]
+
+const imgSrcs = Array.apply(null, Array(5)).map(() => {
+  const min = 50
+  const max = 500
+  const height = Math.floor(Math.random() * (max - min + 1)) + min
+  const width = Math.floor(Math.random() * (max - height + 1)) + height
+  return `https://picsum.photos/${width}/${height}`
+})
+
+const config = {
+  data: imgSrcs,
+  lazyload: true,
+}
+
+const env = process.env.NODE_ENV
+const root = process.env.ROOT_DIR
+const assets = fs.readFileSync(`${root}/dist/webpack-assets.json`)
```

Comment:
What if `${root}/dist/webpack-assets.json` doesn't exist?

Could we make sure `@twreporter/orangutan` would build webpack assets before running  `dev/local-build`?

Maybe in `packages/scrollable-image/Makefile`,
we could call `make build-dist` to tell `orangutan` to build webpack assets in `dist`  folder,
and copy `dist` from `orangutan/dist` to `orangutan/packages/scrollable-image/dist` ? 

Therefore, if we want to `npm run  dev-server`,  we can `make build-dist` first and execute dev-server script.

By the way, we don't need `process.env.ROOT_DIR` anymore.

Reply:
It is a great idea if we want to run a testing server for embedded code. 👍  

Updated.